hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/QuotaUtil.java

```diff
@@ -334,7 +334,8 @@ public static Map<String, UserQuotaState> fetchUserQuotas(final Connection conne
       String user = getUserFromRowKey(key);
 
       if (results[i].isEmpty()) {
-        userQuotas.put(user, buildDefaultUserQuotaState(connection.getConfiguration()));
+        userQuotas.put(user, buildDefaultUserQuotaState(connection.getConfiguration(),
+          EnvironmentEdgeManager.currentTime()));
```

Comment:
can you use the nowTs right above? not sure how much it matters, but it will keep the most parity with how it used to be